www/src/pages/insights.jsx

```diff
@@ -88,6 +120,13 @@ const SummaryUsage = () => {
             accessor: 'name',
           },
           { Header: 'Instance Count', accessor: 'count' },
```

Comment:
Let's add `disableFilters` to this column so there isn't a text filter input field for this column. It doesn't really make sense to want to filter by a specific instance count here.

Reply:
Added `disableFilters`